src/convenience/frameworks.ts

```diff
@@ -231,80 +405,86 @@ const http: FrameworkAdapter = (req, res) => {
-        end: () => res.end(),
+        end: () => response.end(),
         respond: (json) =>
-            res
+            response
                 .writeHead(200, { "Content-Type": "application/json" })
                 .end(json),
-        unauthorized: () => res.writeHead(401).end(WRONG_TOKEN_ERROR),
+        unauthorized: () => response.writeHead(401).end(WRONG_TOKEN_ERROR),
+        handlerReturn: undefined,
     };
 };
 
 /** koa web framework */
-const koa: FrameworkAdapter = (ctx) => ({
-    update: Promise.resolve(ctx.request.body),
-    header: ctx.get(SECRET_HEADER),
+const koa: KoaAdapter = (context) => ({
+    update: Promise.resolve(context.request.body),
+    header: context.get(SECRET_HEADER),
     end: () => {
-        ctx.body = "";
+        context.body = "";
     },
     respond: (json) => {
-        ctx.set("Content-Type", "application/json");
-        ctx.response.body = json;
+        context.set("Content-Type", "application/json");
+        context.response.body = json;
     },
     unauthorized: () => {
-        ctx.status = 401;
+        context.status = 401;
     },
+    handlerReturn: undefined,
 });
 
 /** Next.js Serverless Functions */
-const nextJs: FrameworkAdapter = (req, res) => ({
-    update: Promise.resolve(req.body),
-    header: req.headers[SECRET_HEADER_LOWERCASE],
-    end: () => res.end(),
-    respond: (json) => res.status(200).json(json),
-    unauthorized: () => res.status(401).send(WRONG_TOKEN_ERROR),
+const nextJs: NextAdapter = (request, response) => ({
+    update: Promise.resolve(request.body),
+    header: request.headers[SECRET_HEADER_LOWERCASE],
+    end: () => response.end(),
+    respond: (json) => response.status(200).json(json),
+    unauthorized: () => response.status(401).send(WRONG_TOKEN_ERROR),
+    handlerReturn: undefined,
 });
 
 /** nhttp web framework */
-const nhttp: FrameworkAdapter = (rev) => ({
-    update: rev.body,
+const nhttp: NHttpAdapter = (rev) => ({
+    update: Promise.resolve(rev.body),
     header: rev.headers.get(SECRET_HEADER) || undefined,
     end: () => rev.response.sendStatus(200),
     respond: (json) => rev.response.status(200).send(json),
     unauthorized: () => rev.response.status(401).send(WRONG_TOKEN_ERROR),
+    handlerReturn: undefined,
 });
 
 /** oak web framework */
-const oak: FrameworkAdapter = (ctx) => ({
-    update: ctx.request.body.json(),
-    header: ctx.request.headers.get(SECRET_HEADER) || undefined,
+const oak: OakAdapter = (context) => ({
+    update: context.request.body.json(),
+    header: context.request.headers.get(SECRET_HEADER) || undefined,
     end: () => {
-        ctx.response.status = 200;
+        context.response.status = 200;
     },
     respond: (json) => {
-        ctx.response.type = "json";
-        ctx.response.body = json;
+        context.response.type = "json";
+        context.response.body = json;
     },
     unauthorized: () => {
-        ctx.response.status = 401;
+        context.response.status = 401;
```

Comment:
Is there a reason for making so many unrelated renames? Looks to me like you're deviating from the naming conventions of the respective frameworks, and from their code examples 🤔

Reply:
Just to be clear, this isn't blocking, I was just curious why you think that rolling with non-standard names is better

Reply:
I thought it’d be best to stick to single naming convention, but I can revert this.

Reply:
Ah yeah I think it would be best if every adapter shows idiomatic usage of the respective web framework, you are right that hono did not do this right thus far

Reply:
Done. If I seem to 'miss' some, it's actually because that's actually the naming convention. Only one I am not sure about is Azure.